source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -189,12 +186,14 @@ requirements:
 
       MongoCollection<Flower> collection = database.getCollection("flowers", Flower.class);
 
-Once you have configured the preceding ``MongoCollection`` instance, you can:
+Once you have configured the preceding ``MongoCollection`` instance, you
+can perform :ref:`CRUD operations <java-crud-operations>` with the POJO
+such as:
 
-- save an instance of the POJO to the collection
-- retrieve instances of the POJO from a query on the collection
+- Saving an instance of the POJO to the collection
+- Retrieving instances of the POJO from a query on the collection
 
-The following code shows how you can insert an instance of ``Flower`` into
+The following snippet shows how you can insert an instance of ``Flower`` into
```

Comment:
Issue: "snippet" by itself is less descriptive than "code".
Suggestion: Maybe "code snippet" would be best?